packages/hawtio/src/plugins/camel/index.ts

```diff
@@ -21,6 +21,6 @@ export const camel = () => {
   })
 
   treeProcessorRegistry.add('camel', camelTreeProcessor)
-  // helpRegistry.add('camel', 'Camel', help, 11)
+  helpRegistry.add('camel', 'Camel', help, 11)
```

Comment:
The number passed as the 4th arg should be `13`, which affects order to be displayed in Help view. Currently, single digits are reserved for core helps (Home: 1, Preferences: 2), and the builtin plugins use numbers starting from 11 (Connect: 11, JMX: 12, ...).

The default number is `100`, and we let users to use numbers >= 100 for their own custom plugins. (Then, does it make more sense to change the default to `1000`?)

Reply:
@phantomjinx @mmelko @jsolovjo ^^^ FYI, as probably I haven't explained that yet explicitly.

Reply:
Oh, I see. Nice approach to that. I didn't know what the number should be and I thought it was already commented with the good value. I'll update it.

I don't think we need to increase it to 1000... Having 80+ plugins would be a nice achievement by then :D

Reply:
Why I think maybe we should change default to `1000` is not because we hope to have 80+ more plugins :slightly_smiling_face: but because if it's `100` random custom plugins always take precedence over the custom ones with clear order definitions such as `101`, `102`, ... But probably it's ok to leave it as `100` and let users manage their own plugins order.